database/postgresconn/postgresconn_test.go

```diff
@@ -442,18 +373,6 @@ func TestFailToCreateTableWithoutPermissions(t *testing.T) {
 		if !strings.Contains(e.OrigErr.Error(), "permission denied for schema barfoo") {
 			t.Fatal(e)
 		}
-
-		// re-connect using that x-migrations-table and x-migrations-table-quoted
```

Comment:
removed because we don't support connecting with x-migrations-table and that's all this section was testing, that access is denied when connecting with the x-migrations-table